inc/checks/class-cache-flush.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+
+/**
+ * Detects the number of occurrences of the `wp_cache_flush()` function.
+ */
+class Cache_Flush extends Check {
+
+	public function run() {
+		$result = exec( 'grep -r "wp_cache_flush()" ' . WP_CONTENT_DIR . ' | wc -l' );
```

Comment:
We should use the existing `runcommand\Doctor\Checks\File` filesystem iterator check in here.

Doing so will prevent iterating the filesystem for each check, instead only iterating the filesystem once.

Reply:
@danielbachhuber , I've updated this with iterator, let me know if this is one correctly.